### DISCUSSION TITLE
Recovering Files

### DISCUSSION THREAD

**Discussion by teyplays** [ryohey/signal#224] [Q&A] [answered]
Hi there,

I've just started working with this app and I really like it so far. However, when I go to save files, it freezes for a while before refreshing the browser. When it does so, it clears all the progress I have made. **Is there any way to recover these files?** I was logged in to my account but it doesn't look like there is any place to view in-progress projects. I really hope I haven't wasted a bunch of time on these pieces that won't save.

**Comment by ryohey** [comment, marked as answer]
Hi @teyplays! I feel sorry but files that have not been saved cannot be recovered. Saved files can be found under File > Open after logging in.
Please save files frequently to prevent tragedy. Can you give us more details about the situation so we know what MIDI is causing the freeze?

**Reply by teyplays** [reply]
Thanks for your reply. That's a shame that I can't get them back. But yes, I can definitely give more info.

I am accessing the app via Firefox. The main project that I was having issues with had 2 tracks, 1 with "Synth Effects" (FX 3/crystal) and the other with "Synth Lead" (Lead 3/calliope). I was using a variety of adjustments to the volume, 'expression,' and 'hold pedal' automation settings. Not sure what exactly was causing the problems with saving, but I'll be sure to try to save far more frequently in the future.
